src/Sylius/Behat/Context/Api/Admin/ManagingExchangeRatesContext.php

```diff
@@ -0,0 +1,356 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Component\Currency\Model\CurrencyInterface;
+use Sylius\Component\Currency\Model\ExchangeRateInterface;
+use Sylius\Component\Currency\Repository\ExchangeRateRepositoryInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingExchangeRatesContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $client;
+
+    /** @var ExchangeRateRepositoryInterface */
+    private $exchangeRateRepository;
+
+    public function __construct(ApiClientInterface $client, ExchangeRateRepositoryInterface $exchangeRateRepository)
+    {
+        $this->client = $client;
+        $this->exchangeRateRepository = $exchangeRateRepository;
+    }
+
+    /**
+     * @When I want to add a new exchange rate
+     */
+    public function iWantToAddNewExchangeRate(): void
+    {
+        $this->client->buildCreateRequest('exchange_rates');
+    }
+
+    /**
+     * @When /^I want to edit (this exchange rate)$/
+     */
+    public function iWantToEditThisExchangeRate(ExchangeRateInterface $exchangeRate): void
+    {
+        $this->client->buildUpdateRequest('exchange_rates', $exchangeRate->getId());
+
+        $this->client->addRequestData('sourceCurrency', '/new-api/currencies/EUR');
+        $this->client->addRequestData('targetCurrency', '/new-api/currencies/USD');
```

Comment:
Why do we hardcode these currency codes?

Reply:
I will check immutabling for those two fields in next two steps, Maybe I should check present currency for those fields and change to other?